src/App.js

```diff
@@ -42,10 +45,12 @@ export default class App extends React.Component {
       <>
         <TopBar />
         <main className="container">
-          <section className="sidebar">{/* Recent Donations */}</section>
+          <section className="sidebar">
+            <RecentDonations donations={donations} />
+          </section>
           <section className="">
-            {/* Progress */}
-            {/* Donation Form */}
+            <Progress max={targetAmount} />
+            <DonationForm donations={donations.length +1} />
```

Comment:
This is great, Charlie. Personally though, I would pass only donations.length as the prop and handle the +1 inside of the DonationForm component.